package.json

```diff
@@ -131,5 +131,8 @@
     ]
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
```

Comment:
I'm not sure this ticket requires any changes to `package.json`

Reply:
I'm not sure why that got changed, as I didn't edit anything within this file. Same for yarn.lock